ui/StatusQ/sanity_checker/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(SanityChecker)
+
+find_package(
+  Qt5
+  COMPONENTS Core Quick QuickControls2
+  REQUIRED)
+
+set(QRC_FILES ../src/statusq.qrc ../src/assets.qrc)
+qt5_add_resources(QRC_COMPILED ${QRC_FILES})
```

Comment:
`qt5_add_big_resources` will greatly speed up (re)compile times

Reply:
With `qt5_add_big_resources` changes in qml files wasn't visible after re-running the app on linux. I can change it by the way if you have good remedy for that :)